articles/api/authentication/_multifactor-authentication.md

```diff
@@ -0,0 +1,608 @@
+  "http_method": "POST",
+  "path": "/mfa/challenge",
+  "link": "#multifactor-authentication"
+}) %>
+
+This endpoint lets you request a challenge based on the challenge types supported by the application and the end user. The challenge type indicates the channel or mechanism on which to get the challenge and thus prove possession.
+
+For details on the supported challenge types refer to [Multifactor Authentication and Resource Owner Password](/api-auth/tutorials/multifactor-resource-owner-password).
+
+### Request Parameters
+
+| Parameter        | Description |
+|:-----------------|:------------|
+| `mfa_token` <br/><span class="label label-danger">Required</span> | Token got together with `mfa_required` error for Resource Owner Password flow. |
+| `client_id` <br/><span class="label label-danger">Required</span> | Your application's Client ID. |
+| `client_secret` | Your application's Client Secret. **Required** when the **Token Endpoint Authentication Method** field at your [Client Settings](${manage_url}/#/clients/${account.clientId}/settings) is `Post` or `Basic`. |
+| `challenge_type` | A whitespace-separated list of the challenges types accepted by your application. Accepted challenge types are `oob` or `otp`. Excluding this parameter means that your client application accepts all supported challenge types. |
+| `oob_channel` | **(early access users only)** The channel to use for OOB. Can only be provided when `challenge_type` is `oob`. Accepted channel types are `sms` or `auth0`. Excluding this parameter means that your client application will accept all supported OOB channels. |
+| `authenticator_id` | **(early access users only)** The ID of the authenticator to challenge. You can get the ID by querying the list of available authenticators for the user as explained on __List authenticators__ below. |
+
+### Remarks
+
+- If you already know that OTP is supported by the user and you don't want to request a different factor, you can skip this step an go directly to __Verify with one-time password (OTP)__ below.
+- Auth0 chooses the challenge type based on the application's supported types and types the user is enrolled with.
+- An `unsupported_challenge_type` error is returned if your application does not support any of the challenge types the user has enrolled with.
+- This endpoint does not support enrollment; the user must be enrolled with the preferred method before requesting a challenge.
+- An `unsupported_challenge_type` error is returned if the user is not enrolled.
+- **(early access only)** If the user is not enrolled, you will get a `association_required` error, indicating the user needs to enroll to use MFA. Check __Add an authenticator__ below to see how to proceed.
+
+### More information
+
+- [Multifactor Authentication and Resource Owner Password](/api-auth/tutorials/multifactor-resource-owner-password)
+- [Multifactor Authentication in Auth0](/multifactor-authentication)
+
+## Verify with one-time password (OTP)
+
```

Comment:
None of the /oauth/token endpoints has a `Test with Authentication API Debugger` section. The [Resource Owner Password](https://auth0-docs-content-pr-6008.herokuapp.com/docs/api/authentication#resource-owner-password) does so you should check **if** you can use the Authentication API Debugger can be used to test these endpoints (maybe it can't because MFA input arguments are not in its UI, but you should check either way, it's bad to have API endpoints with no way to test them)


Let me know here if it works or not.

Reply:
All these should also be added in our Postman Collection. Please make sure you add a Trello card in our backlog and copy the link here.

Reply:
I checked the debugger and couldn't find a way to provide the right params.